src/calendar/index.js

```diff
@@ -335,11 +331,10 @@ class Calendar extends Component {
       week.push(this.renderDay(day, id2));
     }, this);
 
-    if (this.props.showWeekNumbers) {
-      week.unshift(this.renderWeekNumber(days[days.length - 1].getWeek()));
-    }
-
-    return (<View style={this.style.week} key={id}>{week}</View>);
+    this.props.showWeekNumbers ? 
```

Comment:
This one doesn't work. You need to return the view with the weeks, there is no if-else here and you're missing a return statement